src/ORM/Queries/SQLSelect.php

```diff
@@ -249,7 +249,7 @@ public function setLimit($limit, $offset = 0)
             throw new InvalidArgumentException("SQLSelect::setLimit() only takes positive values");
```

Comment:
Make this the same as the other exceptions

Reply:
`SQLSelect::setLimit()` does substantially different things than the other `limit` methods. It accepts all sort of weird values like arrays and strings that the other don't.

We could change this error message to something more specific ... but I think it needs to be different than the other methods.